etl/steps/data/garden/technology/2022/internet/__init__.py

```diff
@@ -44,6 +69,39 @@ def make_users_table() -> Table:
     )
     # Filter columns
     table = table[["num_internet_users", "share_internet_users"]]
+    return table.reset_index()
+
+
+def add_regions(table: Table, table_population: Table) -> Table:
+    column_idx = ["country", "year"]
+    # Estimate regions number of internet users
+    regions = [
+        "Europe",
+        "Asia",
+        "North America",
+        "South America",
+        "Africa",
+        "Oceania",
+        "High-income countries",
+        "Low-income countries",
+        "Lower-middle-income countries",
+        "Upper-middle-income countries",
+    ]
+    for region in regions:
+        table = geo.add_region_aggregates(df=table, region=region, aggregations={"num_internet_users": sum})
```

Comment:
At the moment, by default, this function is very strict (and I think we should make it less strict). It checks if all relevant countries have data, and skips aggregates if it finds too many nans. The least strict settings would be:
`countries_that_must_have_data=[],
num_allowed_nans_per_year=None, frac_allowed_nans_per_year=0.99`
This will not care if important countries (in terms of population) have no data, and it imposes that at least 1% of the data for a given year must be not-nan.